app/src/main/java/nerd/tuxmobil/fahrplan/congress/MyApp.kt

```diff
@@ -0,0 +1,81 @@
+package nerd.tuxmobil.fahrplan.congress
+
+import android.app.Application
+import android.util.Log
+
+import androidx.annotation.CallSuper
+
+import org.ligi.tracedroid.TraceDroid
+
+import java.util.*
+
+import info.metadude.android.eventfahrplan.commons.logging.Logging.Companion.get
+import nerd.tuxmobil.fahrplan.congress.repositories.AppRepository.initialize
+import nerd.tuxmobil.fahrplan.congress.utils.ConferenceTimeFrame
+
+class MyApp : Application() {
+
+    enum class TASKS {
+        NONE, FETCH, PARSE, FETCH_CANCELLED
```

Comment:
`FETCH_CANCELLED` seems to be used. Please check the Git history when this happened and mention it in your commit message.

Reply:
Actually, it seems like it was never used 😄 I searched a bit but just found that it was added in 79e35f13 and reformated in 2cac2a38 but wasn't used anywhere 🤔
I'll remove it now.